l10n_it_fatturapa_out_rc/tests/test_e_invoice_out_rc.py

```diff
@@ -46,7 +48,7 @@ def set_sequence_journal_selfinvoice(self, invoice_number, dt):
 
     def set_bill_sequence(self, invoice_number, dt):
         seq_pool = self.env['ir.sequence']
-        inv_seq = seq_pool.search([('name', '=', 'BILL Sequence')])[0]
+        inv_seq = seq_pool.search([('name', '=', 'Vendor Bills')])[0]
```

Comment:
non è più corretto utilizzare il parametro `limit` impostandolo ad `1`? metti caso che la search non ritorni nulla ti salterebbe un `index out of range`